src/gpg/verify.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import gpg
+import time
+
+
+def verify(signature_file, filename):
+    """
+    verify the signature
+    signature_file has the detached signature
+    filename has the name of the file which is signed
+    """
+    
+    c = gpg.Context()
+    
+    try:
+        _, result = c.verify(open(filename), open(signature_file))
+        verified = True
+    except gpg.errors.BadSignatures as e:
+        verified = False
+        print(e)
+
+    if verified is True:
+        for i in range(len(result.signatures)):
```

Comment:
please use explanatory variable names.

Reply:
Actually in Python we can write it like this: `for signature in result.signatures:` which is a bit more elegant in my opinion. I don't have experience with Python, but I know this much.

Reply:
Cool, I'll correct this.